tests/network/bgp/README.md

```diff
@@ -0,0 +1,31 @@
+## BGP Test Infra Overview
+
+BGP tests verify connectivity between the CUDN VM and the external network using BGP.
+
+### External BGP Router
+
+An external BGP router is required for real-world BGP connectivity, as customers typically have a router outside the
+cluster. For testing, the router is implemented as a pod running FRR within the cluster, serving the same role as an
+external router. The external BGP router (a pod in this case) must be on the same network as the cluster nodes
+to enable direct BGP sessions.
+
+### Cluster Requirements
+
+The current implementation requires:
+- Each node must have at least two NICs.
+- All secondary NICs must be connected to the same VLAN as the main IP of the `br-ex` interface.
+
+### Environment Variables
+
+Set these before running tests:
+
+- `BR_EX_VLAN_TAG`: the VLAN number of the `br-ex` main IP.
```

Comment:
The name still does not compute well for me.
I think you can define many secondary networks to work with the br-ex bridge, so this is not reflecting that we are talking about the node network.

The name should be direct and not mention br-ex. The VLAN ID is for the node network access, which in practice is available through secondary NICs.

Therefore, lets name it with something like `NODE_NETWORK_VLAN_TAG`, and not mention br-ex as it may be confusing. You can call it "primary node network" of you find it helpful.